website/client/src/mixins/spells.js

```diff
@@ -34,10 +34,11 @@ export default {
         return null;
       }
 
+      if (this.user.stats.lvl < spell.lvl) return this.text(this.$t('notEnoughLvl'));
```

Comment:
It's not even necessary to throw an error here, methinks--the not-yet-unlocked skills are quite visually distinct from the unlocked ones, so swallowing the click silently is still intuitive.

I could see hiding the details popover for skills that aren't unlocked yet, but that needn't be in scope for this PR.

Reply:
How do you want me to handle the i18n code `return this.text(this.$t('notEnoughLvl'));`? Would u like me to leave it in as a comment for future reference, or do u want me to cut it out?

Reply:
OK to remove it!

Reply:
Got it. Issues is now resolved.